### PR TITLE
REMReM publish misbehaves when overloaded

### PR DESCRIPTION
**Pull request by Thandava-Yelugu**
### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Thandava Krishna Yelugu thandava.yelugu@ericsson.com

### CODE REVIEW

File: publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java [hunk 1]
```diff
@@ -17,8 +17,10 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.HashMap;
+//import java.util.ConcurrentReferenceHashMap;
```
Comment (Surya-Kolli) [inline publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java:20]: Remove this commented line

File: publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java [hunk 2]
```diff
@@ -27,6 +29,7 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 import org.springframework.util.CollectionUtils;
+//import org.springframework.util.ConcurrentReferenceHashMap;
```
Comment (Surya-Kolli) [inline publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java:32]: remove this line

**Review by Surya-Kolli — COMMENTED** [review]
Add the pom.xml file bychanging the publish version

File: publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java (hunk 1 above)
Comment (Thandava-Yelugu) [reply publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java]: ok

**Review by SantoshNC68 — APPROVED** [review]

**Review by Surya-Kolli — APPROVED** [review]

**Review by sudharshan-bandaru — APPROVED** [review]

**Pull request closed without merging**